ngraph/frontend/onnx_editor/include/onnx_editor/editor_types.hpp

```diff
@@ -60,5 +60,77 @@ namespace ngraph
         ///            OutputEdge(5, "out1")
         ///            OutputEdge(5, "out2")
         using OutputEdge = Edge<EdgeType::OUTPUT>;
+
+        /// \brief Specifies a single node input by the name or index.
+        ///        For a node test_node, with 3 inputs:
```

Comment:
Isn't a newline required to end the `\brief` section?
```suggestion
        ///
        ///        For a node test_node, with 3 inputs:
```